adam/experiment/observer.py

```diff
@@ -101,6 +101,11 @@ class CandidateAccuracyObserver(
     Provide an accuracy score.
     """
     name: str = attrib(validator=instance_of(str))
+
+    # these params allow the accuracy to be written out to a text file at each step which is helpful for graphing for
+    # experiments such as the gaze ablation where we want to compare accuracy
+    accuracy_to_txt: bool = attrib(default=False)
+    txt_path: str = attrib(validator=instance_of(str), default="out.txt")
```

Comment:
We'd probably prefer this default to add these records into the experiments output folder where the `index.html` is being logged to for consistency in where to find the outputs for an experiment.

Reply:
Makes sense, I'll update this!